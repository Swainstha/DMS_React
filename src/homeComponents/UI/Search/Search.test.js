import testSnapFunction from '../../../util/testSnapFunction';
import Search from './Search'
import React from 'react';


testSnapFunction("<Search />", "Snapshot test for Search",<Search /> )