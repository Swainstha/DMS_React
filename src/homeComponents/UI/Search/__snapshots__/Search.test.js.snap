// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> Snapshot test for Search 1`] = `ShallowWrapper {}`;
